test/unit/widgets/error.test.ts

```diff
@@ -5,11 +5,17 @@ import { errorMessageBlock } from '../../../src/widgets/error'
 jest.mock('solid-auth-client')
 const dom = new JSDOM('<!DOCTYPE html><p>Hello world</p>').window.document
 
-describe('button', () => {
+describe('errorMessageBlock', () => {
   it('exists', () => {
     expect(errorMessageBlock).toBeInstanceOf(Function)
   })
-  it('runs', () => {
-    expect(errorMessageBlock(dom, '', undefined)).toBeTruthy()
+  it('creates an error message div', () => {
+    expect(errorMessageBlock(dom, 'my error message', undefined)).toMatchSnapshot()
```

Comment:
Don't need to set the third parameter when it's undefined (that's the default behavior, after all).